arm-logic/2016-06-01/swagger/logic.json

```diff
@@ -1767,6 +1806,9 @@
         ],
         "operationId": "Maps_CreateOrUpdate",
         "description": "Creates or updates an integration account map.",
+		"x-ms-examples": {
```

Comment:
make sure the indentation is 2 spaces and not tabs. If you are using VSCode, then replace tabs with 2 spaces and press Alt + Shift  + F for auto formatting.